db/db_test.cc

```diff
@@ -5314,41 +5314,40 @@ TEST_F(DBTest, DynamicMiscOptions) {
 #endif  // ROCKSDB_LITE
 
 TEST_F(DBTest, L0L1L2AndUpHitCounter) {
+  const int kNumKeysPerDb = 30000;
+  const int kNumLevels = 3;
+  const int kNumKeysPerLevel = kNumKeysPerDb / kNumLevels;
```

Comment:
Nit: It might be better to defined kNumKeysPerDb in terms of kNumKeysPerLevel/Levels.  The division requires integer math, whereas multiplication would not.

Reply:
Done.